dev/io.openliberty.io.smallrye.openapi.jaxrs.3.1/.project

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<projectDescription>
+	<name>io.openliberty.io.smallrye.openapi.jaxrs.3.1</name>
```

Comment:
I think that putting the version at the end is misleading. It isn't jaxrs 3.1, it is smallrye openapi 3.1.
```suggestion
	<name>io.openliberty.io.smallrye.openapi3.jaxrs</name>
```